Makefile

```diff
@@ -0,0 +1,25 @@
+SHELL := /bin/bash
+.DEFAULT_GOAL := all
+DOCKER := docker
+TAG ?= $(shell cat VERSION)
+PACKAGE ?= talos-pxe
+TEST_PATTERN ?= "TestLogInfo"
+ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+
+
+.PHONY: unittest-local unittest build-unittest
+
```

Comment:
Remember, if a Makefile is present, a user will want to run `make` and either get info what to do or make it produce something, so don't forget a default target/all.

Reply:
ok